src/version.h

```diff
@@ -0,0 +1,12 @@
+#ifndef VERSION_H
+#define VERSION_H
+
+#define		VERSION_MAJOR		2
+#define		VERSION_MINOR		0
+#define		VERSION_PATCH		0
+#define		VERSION_CANDIDATE	1			// Optional
+#define		VERSION_TYPE		"Alpha "		// RC, Alpha, Beta, etc.
```

Comment:
version here is a placeholder...

arguably this could just reflect the previous 1.4 version.

given that we have added a bunch of new commands, a 1.5 base version may be justified.  similarly it is likely that we will add in the perf PR soon too, which also justifies at least a minor version bump

the alternative argument is that we're moving to a different versioning system, and that is enough to justify a major version bump